Expensify/App
# Handle popover showing from side only in sidebar screen

### Details

The changes made are to support showing from the side the `popover` (FAB options) on iPad.


### Fixed Issues

$ https://github.com/Expensify/App/pull/6278
$ https://github.com/Expensify/App/issues/6520
$ https://github.com/Expensify/App/issues/6519
$ https://github.com/Expensify/App/issues/6522

### Tests


1. Press FAB, options should appear from bottom(iPhone), options should appear from side(iPad).
2. Enter a conversation, press the `+` icon and press `Add attachment`, the next `popover` should appear from bottom.
3. Press emoticons button, `popover` should appear from the bottom and centered from the sides on the screen.
4. Navigate to settings, press avatar to edit profile.
5. Press avatar to change image.
6. Press `Upload photo`, next `popover` should appear from bottom.



### QA Steps


1. Press FAB, options should appear from bottom(iPhone), options should appear from side(iPad).
2. Enter a conversation, press the `+` icon and press `Add attachment`, the next `popover` should appear from bottom.
3. Press emoticons button, `popover` should appear from the bottom and centered from the sides on the screen.
4. Navigate to settings, press avatar to edit profile.
5. Press avatar to change image.
6. Press `Upload photo`, next `popover` should appear from bottom.


### Tested On


### Screenshots

#### iPhone

https://user-images.githubusercontent.com/53409604/145513505-1c4f1616-b48f-4a07-9d0a-9e5d569005d4.mp4



#### iPad

https://user-images.githubusercontent.com/53409604/145513652-7bb92bea-8a09-46d4-a2b2-7128144cf57d.mp4


#### Android
https://user-images.githubusercontent.com/53409604/145513692-deba593a-886b-48fd-ae3d-c018d7756e56.mov


#### Web

https://user-images.githubusercontent.com/53409604/145513768-4f63e112-6d99-4055-8286-cd60718737c1.mov
